content/admin/convert-account.md

```diff
@@ -6,9 +6,14 @@ aliases:
 - /docker-hub/convert-account/
 ---
 
+> **Tip**
+>
+> You can convert an account with a Docker Personal subscription to an organization. If you need to convert an account with a Docker Pro subscription, [contact support](https://hub.docker.com/support/contact).
```

Comment:
**nitpick:** maybe adding a link (https://account.docker.com/) where to convert an account 🤷🏻‍♂️

Reply:
There's instructions later on in the doc for where to go to convert (this is still in Hub for now).